src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs

```diff
@@ -283,73 +284,6 @@ private JObject TryAddBundleConfiguration(JObject content, string bundleId, stri
                 }
                 return content;
             }
-
-            internal static string SanitizeHostJson(JObject hostJsonObject)
```

Comment:
I needed a JObject recursive sanitizer, so I generalized the below code since only a single portion of it was specific to host.json (the list of properties to sanitize).